app/components/form_components/schools_autocomplete/script.js

```diff
@@ -0,0 +1,68 @@
+import accessibleAutocomplete from 'accessible-autocomplete'
+
+const $form = document.querySelector('[data-module="app-schools-autocomplete"]')
+const idElement = document.getElementById('lead-school-id')
+
+let statusMessage = null
+
+const findSchools = (query, populateResults) => {
+  statusMessage = 'Loading...'
+
+  window.fetch(`/api/schools?query=${query}&lead_school=true`)
+    .then(response => {
+      return response.json()
+    })
+    .then(data => {
+      if (data === undefined) {
+        return
+      }
+
+      const schools = query ? data.schools : []
+
+      if (schools.length === 0) {
+        statusMessage = 'No results found'
+      }
+
+      populateResults(schools)
```

Comment:
No, return school here

then its thenable

```suggestion
return school;
).then( populateResults)
```

Reply:
All done 👍